src/CMakeLists.txt

```diff
@@ -8,27 +8,53 @@
 
 include_directories(.)
 
+# Added as part of the merge with build only infrastructure:
 add_subdirectory(common)
+
+if(NOT INFRASTRUCTURE_ONLY)
 add_subdirectory(apps)
+add_subdirectory(apps-kokkos)
```

Comment:
I think these should be guarded with ENABLE_KOKKOS

Reply:
> I think these should be guarded with ENABLE_KOKKOS

Hi David, if I am understanding your suggestion, guarding all things Kokkos in this file will require significant reorganization of this file.  Is that what you would like for me to do, i.e., cleanly separate RAJAPerf Suite content from Kokkos content?